File: 卡码/KamaCoder/Go/test06.go
// 6. A+B问题VIII
package main

import (
	"fmt"
)

func main(){
	for {
		var n int
		if _,err:=fmt.Scan(&n);err!=nil {
			return
		}
		for i:=0;i<n;i++ {
			var m int
			fmt.Scan(&m)
			var sum int
			for j:=0;j<m;j++ {
				var tmp int
				fmt.Scan(&tmp)
				sum+=tmp
			}
			fmt.Println(sum)
			if i!=n-1 {
				fmt.Println()
			}
		}
	}
}

File: 卡码/KamaCoder/Js/test06.js
//6. A+B问题VIII
const rl=require("readline").createInterface({input:process.stdin});
const iter=rl[Symbol.asyncIterator]();
const readline=async ()=>(await iter.next()).value;

async function main(){
  while(line=await readline()){
    const n=parseInt(line);
    for(let i=0;i<n;i++){
      let sum=0;
      const [m,...nums]=(await readline()).split(" ").map(Number);
      console.log(nums.reduce((item,prev)=>item+prev,0));
      if(i!=n-1) console.log();
    }
  }
}

main();